src/main/java/in/pune/royforge/eledgerapi/data/repo/ITransactionLogRepository.java

```diff
@@ -5,5 +5,6 @@
 import in.pune.royforge.eledgerapi.data.entity.TransactionEntity;
 
 public interface ITransactionLogRepository extends PagingAndSortingRepository<TransactionEntity, String> {
+	public Iterable<TransactionEntity> findByLenderId(String lenderId);
```

Comment:
change return type Iterable<TransactionEntity> ---> List<TransactionEntity> and then use it.